### PR TITLE
Alert on group disappear

### PR DESCRIPTION
**Pull request by jadev-dev**
add support for "alert on group disappear" option

### CODE REVIEW

File: plugins/module_utils/kibana.py [hunk 1]
```diff
@@ -207,6 +207,7 @@ def format_alert_params(self):
       formatted_params['filterQuery'] = self.module.params.get('filter_query')
     if self.module.params.get('group_by'):
       formatted_params['groupBy'] = self.module.params.get('group_by')
+      formatted_params['alertOnGroupDisappear'] = self.module.params.get('alert_on_group_disappear')
```
Comment (maclin-masterson) [inline plugins/module_utils/kibana.py:210]: Do we expect `alert_on_group_disappear` to always be provided to the module when `group_by` exists? If not, I would argue for a nested if statement here to check for `alert_on_group_disappear` so we aren't including null values in the parameters.

File: plugins/module_utils/kibana.py [hunk 2]
```diff
@@ -547,4 +548,4 @@ def get_agent_list(self):
         agent_list_result['list'].append(agent_list_page['list'][agent_no])   
         agent_no = agent_no + 1
       page_number = page_number + 1
-    return agent_list_result
\ No newline at end of file
+    return agent_list_result
```
Comment (maclin-masterson) [inline plugins/module_utils/kibana.py:551]: No newline at end of file

File: plugins/module_utils/kibana.py (hunk 1 above)
Comment (jadev-dev) [reply plugins/module_utils/kibana.py]: Added check for `None` and supplied a default value of `False`. The default when this value isn't specified is `True`, which is almost never desirable, as groups tend to change in normal operation and the resulting notifications are worded in a confusing way. (Basically, something like Low Disk Space would trigger if a previously mounted drive was unmounted, and would send a notification with the wording "Low Disk Space")

**Comment by maclin-masterson** [comment]
LGTM

**Review by maclin-masterson — APPROVED** [review]

**Pull request merged**